src/reanimated2/core.ts

```diff
@@ -383,9 +383,11 @@ if (!NativeReanimatedModule.useOnlyV1) {
         info: runOnJS(capturableConsole.info),
       };
       _setGlobalConsole(console);
-      global.performance = {
-        now: global._chronoNow,
-      };
+      if (!runOnJS(shouldBeUseWeb)) {
```

Comment:
```suggestion
      if (!runOnJS(shouldBeUseWeb)()) {
```

Reply:
💯

Reply:
💯. Also, `shouldBeUseWeb` probably is not the right function to use here. Looks like @piaskowyk assigned it to himself. Probably he'll solve this